hnn_core/drives.py

```diff
@@ -299,11 +299,10 @@ def _drive_cell_event_times(drive_type, dynamics, tstop, target_type='any',
     # brute force remove non-zero times. Might result in fewer vals
     # than desired
     # values MUST be sorted for VecStim()!
-    if len(event_times) > 0:
-        event_times = event_times[np.logical_and(event_times > 0,
-                                                 event_times <= tstop)]
-        event_times.sort()
-        event_times = event_times.tolist()
+    event_times = event_times[np.logical_and(event_times > 0,
```

Comment:
is the logic that this will still work as intended on empty lists?

Reply:
Yes! The initial goal was to just make sure `event_times` was getting processes the same way every time regardless of its contents.